src/applications/pact-act/containers/questions/radiation/Radiation-2-3-B.jsx

```diff
@@ -71,11 +71,11 @@ const mapDispatchToProps = {
 
 Radiation23B.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setRadiation23B: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.